.travis.yml

```diff
@@ -15,7 +15,9 @@ cache:
   directories:
     - node_modules
 before_install:
-  - npm install -g @lhci/cli@0.4.x
+  - npm install -g @lhci/cli@0.9.x
+install:
+  - npm install --legacy-peer-deps
```

Comment:
should this use `npm ci` instead of npm install?

Reply:
i don't know if travis actually caches anything so im not sure there is a difference between ci and install when it comes to travis

Reply:
updated just in case